service/history/nDCTransactionMgr.go

```diff
@@ -149,6 +159,7 @@ var _ nDCTransactionMgr = (*nDCTransactionMgrImpl)(nil)
 func newNDCTransactionMgr(
 	shard ShardContext,
 	historyCache *historyCache,
+	clientBean client.Bean,
```

Comment:
client bean can be get from shard.GetService().GetClientBean()
so remove `clientBean client.Bean,`

Reply:
done